listenbrainz/webserver/static/css/brainzplayer.less

```diff
@@ -240,7 +250,18 @@
 		}
 		&:hover{
 			height: @progress-bar-hover-height;
-			top: -@progress-bar-height - (@progress-bar-hover-height - @progress-bar-height)/2
+			top: -@progress-bar-hover-height;
+		}
+
+		.progress-tooltip {
```

Comment:
We are going to want the tooltip to appear above the rest of the content we have on the page, like the youtube layer for example.
Currently, the tooltip appears behind the youtube player, which also blocks mouse hovering.

This means changing the z-index property of a couple of elements, and preventing the youtube window from overlapping the player (because of some CSS quirks I don't think we can resolve): 

- The `.youtube-wrapper`'s `z-index` should go from 10 to 4
- `#brainz-player .progress`'s `z-index` should go from 2 to 5
- in YoutubePlayer.tsx, the [Draggable element's bottom bound](https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz/webserver/static/js/src/brainzplayer/YoutubePlayer.tsx#L409) should be set to `-draggableBoundPadding`:
<img width="225" alt="image" src="https://user-images.githubusercontent.com/6179856/170990408-189e8575-7047-4ab6-935b-cee06703f577.png">

The result of all that is the tooltip hovers over the youtube window:
<img width="401" alt="image" src="https://user-images.githubusercontent.com/6179856/170990893-953fe9b1-4fac-426a-a9a4-f57ca52e47f4.png">

Reply:
Awesome! that bugged me too.